core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTest.java

```diff
@@ -1090,6 +1123,48 @@ void testWildcard() throws Exception {
         }
     }
 
+    @Test
+    void testMultiParams() throws Exception {
+        try (CloseableHttpClient hc = HttpClients.createMinimal()) {
```

Comment:
Use Armeria client?

Reply:
Could you explain more? In the `AnnotatedServiceTest`, almost all of the tests including assertions (`testBody`) are using `CloseableHttpClient`.

Reply:
Please ignore the comment. I guess the tests were added before `WebClient` was made.
We can clean up the legacy code later. That is not related to this PR.